python/cugraph/cugraph/community/subgraph_extraction.py

```diff
@@ -59,7 +60,11 @@ def subgraph(G, vertices):
     G, isNx = ensure_cugraph_obj_for_nx(G)
 
     if not G.edgelist.weights:
-        raise RuntimeError("input graph must be weighted")
+        warning_msg = (
```

Comment:
C++ induced_subgraph does not require weights.  Not sure we should require that in python.

Reply:
Correct. I just double checked